RSSTimeFormatter/RealDateTimeFormatter.cs

```diff
@@ -65,30 +65,31 @@ public string PrintTimeStampCompact(double time, bool days = false, bool years =
 		}
 		public string PrintTime(double time, int valuesOfInterest, bool explicitPositive)
 		{
-			// This is a downright strange and confusing method but as I understand it

```

Comment:
If the comment is erroneous, let's update the comment to be correct, but leave it in as this is such a confusing piece of code it helps to have the comment for future head-scratching.

Reply:
Added explanation in comment